sui_core/src/authority/authority_store.rs

```diff
@@ -54,11 +59,11 @@ pub struct SuiDataStore<const ALL_OBJ_VER: bool> {
     /// by a specific user, and their object reference.
     owner_index: DBMap<(SuiAddress, ObjectID), ObjectRef>,
 
-    /// This is map between the transaction digest and signed transactions found in the `transaction_lock`.
+    /// This is map between the transaction digest and transactions found in the `transaction_lock`.
     /// NOTE: after a lock is deleted the corresponding entry here could be deleted, but right
-    /// now this is not done. If a certificate is processed (see `certificates`) the signed
+    /// now this is not done. If a certificate is processed (see `certificates`) the
```

Comment:
Might be good to note here that transactions can be signed or not depending on the type of S.